src/patternfly/components/Card/card.scss

```diff
@@ -17,8 +17,17 @@
   --pf-c-card--m-compact--child--PaddingLeft: var(--pf-global--spacer--md);
   --pf-c-card--m-compact--c-divider--child--PaddingTop: var(--pf-global--spacer--md);
   --pf-c-card--m-compact__title--not--last-child--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-card--m-large__body--FontSize: var(--pf-global--FontSize--lg);
+  --pf-c-card--m-large__footer--FontSize: var(--pf-global--FontSize--lg);
+  --pf-c-card--m-large--first-child--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-card--m-large--child--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-card--m-large--child--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-card--m-large--child--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-card--m-large--c-divider--child--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-card--m-large__title--not--last-child--PaddingBottom: var(--pf-global--spacer--lg);
```

Comment:
Looking at these designs, it looks like the header/title should have the `--xl` spacer.

<img width="255" alt="Screen Shot 2021-01-12 at 2 12 05 PM" src="https://user-images.githubusercontent.com/35148959/104367300-3a276580-54e0-11eb-92a7-e71e4cc8dd05.png">

In the card, the title and header are somewhat synonymous as far as the spacing is concerned. The header is just used to wrap the title if you have other things you want to put adjacent to the title. In the examples in the design, there is a "title" but that looks to just be a generic title, not necessarily the card's title element. Do you see it differently?

Reply:
@mcoker Sorry for not noticing that! Thank you, will update.